otter/integration/tests/test_convergence.py

```diff
@@ -425,6 +431,87 @@ def _remove_metadata(self, ids, rcs):
             NovaServer(id=_id, pool=self.helper.pool).update_metadata({}, rcs)
             for _id in ids]).addCallback(lambda _: rcs)
 
+    def test_a(self):
+        return self.test_clb_pending_delete_on_scale_up()
+
+    def test_clb_pending_delete_on_scale_up(self):
+        """
+        Simulate getting a 422 on scale up from CLB when in PENDING_DELETE
+        which should cause the group to error.
+        - Create a group with non-zero min servers and a CLB (or more)
+        - Set mimic to return 422 and state PENDING_DELETE on addition to CLB
+        - Attempt to scale up
+        - Group goes into error state since it cannot take action
+
+        Prereq: MUST be able to use Mimic to simulate incorrect CLB behavior
```

Comment:
We can add the `@skip_if(not_mimic, "This requires Mimic for error injection.")` decorator instead of the comment, which will skip if the environment variable for mimic isn't set.  :)